scripts/eject.js

```diff
@@ -104,6 +104,11 @@ prompt('Are you sure you want to eject? This action is permanent. [y/N]', functi
   });
   delete hostPackage.scripts['eject'];
 
+  // explicitly specify ESLint config path for editor plugins
+  hostPackage.eslintConfig = {
+    extends: "./config/eslint.js",
```

Comment:
I actually think it won't be, Node fs functions treat / in platform independent way. I wrote some unnecessary path.joins in the code before I learned this.